sql/core/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveDefaultColumnsSuite.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.analysis
```

Comment:
Hi, @dtenedor and @gengliangwang . I have a question.

Although I understand this test suite provide a test coverage for `org.apache.spark.sql.catalyst.analysis.ResolveDefaultColumns`, it doesn't mean this test suite is belong to `org.apache.spark.sql.catalyst.analysis` package. This test suite exists in `sql` module and alone in this directory

```
$ tree sql/core/src/test/scala/org/apache/spark/sql/catalyst
sql/core/src/test/scala/org/apache/spark/sql/catalyst
└── analysis
    └── ResolveDefaultColumnsSuite.scala
```

Is this intentional?

Reply:
Hi @dongjoon-hyun I don't think this is intentional, we could move the `ResolveDefaultColumnsSuite` to `org.apache.spark.sql` package. What do you think? If you want me to do this, I can prepare a PR.

Reply:
Thank you. I made a PR.
- https://github.com/apache/spark/pull/41520